object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,26 @@
+/* exported defaults */
+
+// create storage for a test if a key of the source argument is not defined in the target argument and set as true
+// look at each key of the source argument
+// - look at each key of the target argument
+//  - if current key of source is equal to current key of target and current key of target is not undefined
```

Comment:
This point of pseudocode is **_very_** tech jargony. What is the step you are actually doing?